notebooks/nzh-20230417-00-embeddings-classifier.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.                                         explode_preds.col["label"].alias("label"),
</pre><p>The shorthand to get the individual columns is to alias the previous exploded field (<code>F.explode(...).alias("col")</code>, and to do <code>"col.*"</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>